src/lightning_app/structures/list.py

```diff
@@ -52,23 +50,21 @@ def __init__(self, *items: T):
 
         self._name: t.Optional[str] = ""
         self._last_index = 0
-        self._backend: Optional[Backend] = None
+        self._backend: t.Optional[Backend] = None
```

Comment:
why import with `t.` and not `from typing import ...`?

Reply:
Because we already have `import typing as t` in there

Reply:
no I see the weird typing hack :D